src/coreclr/jit/hwintrinsiclistxarch.h

```diff
@@ -847,6 +873,8 @@ HARDWARE_INTRINSIC(SSE2,            UCOMISD,
 HARDWARE_INTRINSIC(SSE41,           PTEST,                                      16,              2,     {INS_ptest,             INS_ptest,              INS_ptest,              INS_ptest,              INS_ptest,              INS_ptest,              INS_ptest,              INS_ptest,              INS_invalid,            INS_invalid},           HW_Category_SimpleSIMD,             HW_Flag_NoRMWSemantics|HW_Flag_NoEvexSemantics)
 HARDWARE_INTRINSIC(AVX,             PTEST,                                       0,              2,     {INS_ptest,             INS_ptest,              INS_ptest,              INS_ptest,              INS_ptest,              INS_ptest,              INS_ptest,              INS_ptest,              INS_vtestps,            INS_vtestpd},           HW_Category_SimpleSIMD,             HW_Flag_NoRMWSemantics|HW_Flag_NoEvexSemantics)
 
+HARDWARE_INTRINSIC(AVX512F,         MoveMaskSpecial,                            64,              1,     {INS_invalid,           INS_invalid,            INS_invalid,            INS_invalid,            INS_movd,               INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid,            INS_invalid},           HW_Category_SIMDScalar,            HW_Flag_BaseTypeFromFirstArg|HW_Flag_NoContainment|HW_Flag_SpecialCodeGen|HW_Flag_NoRMWSemantics)
```

Comment:
Wondering why do you need `MoveMaskSpecial`? Can't you handle it using `NI_Vector512_ExtractMostSignificantBits`?

Reply:
The other `MoveMask` instructions are modeled directly and we'll, long term, want `MoveMask` to be the same.

So having it as `MoveMaskSpecial` will presumably make it simpler to transition once the proper `AVX512F_MoveMask` support is added in the future.